mlflow/utils/autologging_utils/__init__.py

```diff
@@ -543,4 +543,21 @@ def should_log(self):
                 self._parent.allow_children and self._parent.clazz != self.clazz
             )
 
+        @staticmethod
+        def is_active():
+            return len(_TrainingSession._session_stack) == 0
```

Comment:
is_active() should be `True` if there is an active sklearn autologging session. Can we adjust the logic here and change the usages to check `not _TrainingSession.is_active()` instead of `_TrainingSession.is_active()`?

```suggestion
            return len(_TrainingSession._session_stack) != 0
```